test/parser/test_captcha_hit.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from harambe.errors import CaptchaError, HarambeException
+
+
+def simulate_captcha_scenario():
+    # Simulating a CAPTCHA hit scenario
+    raise CaptchaError()
+
+
+def test_captcha_hit_exception():
+    # Testing if HarambeException is raised when expected
+    with pytest.raises(HarambeException, match="CAPTCHA was hit."):
```

Comment:
Don't really care to match the string here. Will make the test brittle if we decide to change the message. We just care that it will be captured as a HarambeError
```suggestion
    with pytest.raises(HarambeException):
```